workflows/detaxizer.nf

```diff
@@ -230,13 +239,13 @@ workflow DETAXIZER {
 
     // preparation of the channels
     ch_prepare_summary_kraken2 = KRAKEN2_KRAKEN2.out.classified_reads_assignment.join(ISOLATE_IDS_FROM_KRAKEN2_TO_BLASTN.out.classified)
-    ch_prepare_summary_kraken2 = ch_prepare_summary_kraken2.map { 
-        meta, path1, path2 -> 
-        [ ['id': meta.id ], [ path1, path2 ] ] 
+    ch_prepare_summary_kraken2 = ch_prepare_summary_kraken2.map {
+        meta, path1, path2 ->
+        [ ['id': meta.id ], [ path1, path2 ] ]
```

Comment:
Why only taking the id? Is there a reason not to use other namespaces of the metamap here?

Reply:
Worked on it, thanks for the hint!